src/main/java/de/jenvy/haitatsu/packet/AbstractHaitatsuPacketFactory.java

```diff
@@ -53,7 +56,7 @@ public HaitatsuPacket getPacket(int packetId) {
         } catch (InstantiationException | IllegalAccessException | InvocationTargetException | NoSuchMethodException e) {
             e.printStackTrace();
         }
-        return null;
```

Comment:
We should avoid any null references, I'm now really used to writing code null-safe and we can make use of a few new Java features here.